Description: Upstream changes introduced in version 2.0.0-10~lucidubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 bumblebee (2.0.0-10~lucidubuntu1) lucid; urgency=high
 .
   * Fixed missing REPLACEMONITOR in configuration script
 .
 The person named in the Author field signed this changelog entry.
Author: Martin Juhl (MrMEEE) <mj@casalogic.dk>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- bumblebee-2.0.0.orig/install-files/bumblebee-configuration
+++ bumblebee-2.0.0/install-files/bumblebee-configuration
@@ -282,6 +282,8 @@ if [ -f /usr/local/bin/bumblebee-disable
 fi
 fi
 
+sed -i 's/REPLACEWITHCONNECTEDMONITOR/'$CONNECTEDMONITOR'/g' /etc/X11/xorg.conf.nvidia
+
 if [ $CONFIGMODE != "MANUAL" ]; then
 if [ $POWERON != "UNAVAILABLE" ]; then
  echo $POWERON | xxd -r -p > /usr/local/bin/bumblebee-enablecard
